dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2572,24 +2840,371 @@
       "x-ms-discriminator-value": "NumericRangeResolution",
       "properties": {
         "rangeKind": {
+          "$ref": "#/definitions/NumericRangeKind"
+        },
+        "minimum": {
+          "type": "number",
+          "format": "double",
+          "description": "The beginning value of  the interval."
+        },
+        "maximum": {
+          "type": "number",
+          "format": "double",
+          "description": "The ending value of the interval."
+        }
+      },
+      "required": [
+        "rangeKind",
+        "minimum",
+        "maximum"
+      ]
+    },
+    "BaseMd": {
```

Comment:
Abbreviations are generally frowned on for various reasons, especially in the client libraries. Can you just call this `BaseMetadata` or `BaseEntityMetadata`?

Reply:
You could also use an `x-ms-client-enum` for that if you don't want to change the model, though the model name appears nowhere in the payload.

Reply:
Changed all to say Metadata